### PR TITLE
[BACKLOG-37966] Toolbar improvements in PUC

### PR DESCRIPTION
**Pull request by AAravind27**
@pentaho/wcag  , Please review for https://hv-eng.atlassian.net/browse/BACKLOG-37966

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 18m 32s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **359**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/data-access-plugin/job/PR-1222/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/controllers/WizardRelationalDatasourceController.java [hunk 1]
```diff
@@ -131,8 +133,9 @@ public void init( final DatasourceModel datasourceModel ) {
     previewResultsTable = (XulTree) document.getElementById( "previewResultsTable" ); //$NON-NLS-1$
     previewResultsTreeCols = (XulTreeCols) document.getElementById( "previewResultsTreeCols" ); //$NON-NLS-1$
     previewLimit = (XulTextbox) document.getElementById( "previewLimit" ); //$NON-NLS-1$
-    editConnectionButton = (XulButton) document.getElementById( "editConnection" ); //$NON-NLS-1$
-    removeConnectionButton = (XulButton) document.getElementById( "removeConnection" ); //$NON-NLS-1$
+    editConnectionButton = (XulToolbarbutton) document.getElementById( "editConnection" ); //$NON-NLS-1$
+    removeConnectionButton = (XulToolbarbutton) document.getElementById( "removeConnection" ); //$NON-NLS-1$
+    addConnectionButton = (XulToolbarbutton) document.getElementById( "addConnection" ); //$NON-NLS-1$
```
Comment (dcleao) [inline core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/controllers/WizardRelationalDatasourceController.java:138]: Why is this new class property, `addConnectionButton`, needed?

**Review by dcleao — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/controllers/WizardRelationalDatasourceController.java (hunk 1 above)
Comment (AAravind27) [reply core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/controllers/WizardRelationalDatasourceController.java:138]: In `main_wizard_panel.xul`, I converted 3 buttons to toolbar buttons. In `WizardRelationalDatasourceController.java`, I could find only 2 of them , so I added `addConnectionButton`. Turns out the addition was not needed.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 19m 3s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core
```
---
:ok_hand: All tests passed!
 
Tests run: **359**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/data-access-plugin/job/PR-1222/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Review by hv-vamsi — APPROVED** [review]

**Pull request merged**